projects/element-ng/landing-page/login-basic/si-login-basic.component.scss

```diff
@@ -0,0 +1,53 @@
+@use 'sass:map';
+
+@use '@siemens/element-theme/src/styles/variables';
+
+:host {
+  --animation-duration: 300ms;
+  --animation-delay: calc(var(--animation-duration) / 2);
+  --ease-out-quint: cubic-bezier(0.22, 1, 0.36, 1);
+  --ease-out-quad: cubic-bezier(0.5, 1, 0.89, 1);
```

Comment:
we usall just use the standard easing function. While we may actually should have our own in general, I guess just using `ease-*`  for now is the better approach